src/pages/People.js

```diff
@@ -0,0 +1,407 @@
+import React from "react"
+import Helmet from "react-helmet"
+import s from "styled-components"
+import { Grid, Cell } from "styled-css-grid";
+import "../pages/styles.scss"
+import { Title, Text, Subtitle} from "../components/shared/Typography"
+import { Navbar, Footer } from "../components"
+
+import ArunMajumdar from '../images/arun_majumdar.jpeg'
+import DennisWoodside from '../images/dennis woodside.jpg'
+import ZhengxiaDou from '../images/zhengxia_dou.jpeg'
+import DavdGalligan from '../images/david_galligan.jpeg'
+import JamesHagan from '../images/james_hagan.jpeg'
+import PeterPsarras from '../images/peter_psarras.jpeg'
+import ArthurBenthem from '../images/arthur_benthem.jpeg'
+import ESwann from '../images/e_swann.jpg'
+import AlexBehar from '../images/alex_behar.jpeg'
+import ShuoYang from '../images/shuo_yang.jpeg'
+import AllyWarson from '../images/ally_warson.jpeg'
+import KimberlyZou from '../images/kimberly_zou.jpeg'
+import RyFisher from '../images/ry_fisher.jpeg'
+import FranzHochstrasser from '../images/franz_hochstrasser.jpeg'
+import CaitlynMcCloskey from '../images/caitlyn_mcCloskey.jpeg'
+import BenMay from '../images/ben_may.jpeg'
+import AkaashPadmanabha from '../images/akaash_padmanabha.jpeg'
+import PEGLogo from '../images/peg.jpeg'
+import SSAPLogo from '../images/ssap.png'
+import PUCPLogo from '../images/pucp.jpeg'
```

Comment:
so originally I imported images this way to leverage Gatsby's image loading code - Gatsby resizes the image at build time and creates a low, mid, and high resolution version of the image. This low resolution image can then be sent to the client initially to create the illusion of the page loading images really quickly...

but now looking at this maybe it's time to stop. Holy. Perhaps you should just host these images in a public folder and then reference the image sources using relative url. LMK if you want to do this/have questions